Update preset-security-policies.md (#9266)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/office-365-security/preset-security-policies.md

To disable the **Standard protection** or **Strict protection** preset security policies while still preserving the existing conditions and exceptions, slide the toggle to **Disabled** ![Toggle Off.](../../media/scc-toggle-off.png). To enable the policies, slide the toggle to **Enabled** ![Toggle On](../../media/scc-toggle-on.png).

### Use PowerShell to review the settings of the preset security policy

> There are two cmdlets that we can use to view these policies:

Reviewer: ```suggestion
There are two cmdlets that you can use to view these policies:
```

> 1. To View Safe Attachments and Safe Link Policies, you can use the below Cmdlet

Reviewer: ```suggestion
1. To view Safe Attachments and Safe Link policies, use the following cmdlet:
```

```
Get-ATPProtectionPolicyRule
```

> 2. To View Hosted Connection Filter, Anti-Phish and Malware Filter you can use the below Cmdlet

Reviewer: ```suggestion
2. To view the Hosted Connection, Anti-Phish, and Malware filters, use the following cmdlet:
```

```
Get-EOPProtectionPolicyRule
```

> 

Reviewer: ```suggestion
```
